lib/bolt/pal.rb

```diff
@@ -43,12 +43,13 @@ def self.load_puppet
       Bolt::ResultSet.include_iterable
     end
 
-    # Create a top-level alias for TargetSpec so that users don't have to
+    # Create a top-level alias for TargetSpec and PlanResult so that users don't have to
     # namespace it with Boltlib, which is just an implementation detail. This
-    # allows TargetSpec to feel like a built-in type in bolt, rather than
+    # allows them to feel like a built-in type in bolt, rather than
     # something has been, no pun intended, "bolted on".
-    def add_target_spec(compiler)
+    def alias_types(compiler)
       compiler.evaluate_string('type TargetSpec = Boltlib::TargetSpec')
+      compiler.evaluate_string('type PlanResult = Boltlib::PlanResult')
```

Comment:
Do we expect people to use PlanResult directly?

Reply:
In functions that run plans perhaps